### PR TITLE
Update classification_and_policy_enforcement.adoc

### PR DESCRIPTION
**Pull request by jnweiger**
Fixes: #1248

Add background and examples for the classification of PDF and images using the exif metadata.
This can be tedious.

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 1]
```diff
@@ -196,6 +212,17 @@ Apart from the concrete examples above, a generalized method to employ document
 - Note down the classification property and value.
 - Repeat the steps for all classification properties and values you want to set up classification rules for in ownCloud.
 
+==== Finding Metadata in PDF or Images
+
+- The command line tool `exiftool` can be used to inspect a file for suitable metadata. 
+- The output of `exiftool -n -q -b -X example.pdf` shows the Metadta XML-Syntax. Use the XML node names to generate XPath queries and XML node data for the Property Value.
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:218]:
```suggestion
- The output of `exiftool -n -q -b -X example.pdf` shows the Metadata XML-Syntax. Use the XML node names to generate XPath queries and XML node data for the Property Value.
```

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 2]
```diff
@@ -117,6 +117,12 @@ The configuration depends on the tools and the classification framework in use.
 
 Administrators can find examples and generalized configuration instructions below.
 
+=== EXIF Header Metadata
+
+PDF and several Image formats (JPEG, PNG, HEIC) support EXIF metadata. For a full list of supported file formats see the documentation of exiftool.
+Support for EXIF was added in version 1.5.0 of the files_classifier app. Please note, that the feature requires the installation of exiftool. (E.g. 
+sudo apt install exiftool)
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:124]:
```suggestion

IMPORTANT: For several file types, the classification relies on the installation of `exiftool`. For details see below.

PDF and several Image formats like JPEG, PNG, HEIC support EXIF metadata. See the documentation for a full list of https://exiftool.org/#supported[supported file formats] of exiftool. Support for EXIF was added in version 1.5.0 of the {oc-marketplace-url}/app/files_classifier[Files Classifier app]. Please note that the feature requires the installation of exiftool. 

[source,bash]
----
sudo apt install exiftool
----
```

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 3]
```diff
@@ -183,6 +195,16 @@ This is how you set up an automated classification and the feature policy in own
...
 
+===== EXIF Metadata
+
+The available elements in EXIF metadata can vary depending how the file was created.
+* Photo cameras may embed the Product name: **Property XPath** = `//IFD0:Model` and e.g. **Property Value** = `Canon EOS 40D`
+* Software creating PDF documents may embed their name: 
+  ** Property XPath** = `//xmp:CreatorTool` and **Property Value** = `Adobe InDesign 15.0 (Macintosh)`
+* Screenshot or graphics software may have e.g.
+  ** Property XPath** = `///PNG:Software` and **Property Value** = `gnome-screenhot`
+  ** Property XPath** = `//PNG:Comment` and **Property Value** = `Created with GIMP`
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:206]:
```suggestion
The available elements in EXIF metadata can vary depending how the file was created.
* Software creating PDF documents may embed their name: +
**Property XPath** = `//xmp:CreatorTool` and **Property Value** = `Adobe InDesign 15.0 (Macintosh)`

* Photo cameras may embed the Product name: +
**Property XPath** = `//IFD0:Model` and e.g. **Property Value** = `Canon EOS 40D`

* Screenshot or graphics software may have: +
**Property XPath** = `///PNG:Software` and **Property Value** = `gnome-screenhot` +
**Property XPath** = `//PNG:Comment` and **Property Value** = `Created with GIMP`
```

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 4]
```diff
@@ -196,6 +221,17 @@ Apart from the concrete examples above, a generalized method to employ document
...
+==== Finding Metadata in PDF or Images
+
+- The command line tool `exiftool` can be used to inspect a file for suitable metadata. 
+- The output of `exiftool -n -q -b -X example.pdf` shows the Metadata XML-Syntax. Use the XML node names to generate XPath queries and XML node data for the Property Value.
+- The metadata may use XML Namespaces. In `xmp:CreatorTool` the namespace is `xmp`. 
+- We can use some more complex XPath expression to match the CreatorTool e.g. in any namespace or by prefix:
+  ** **Property XPath** = `//*[local-name()='Comment']` and **Property Value** = `Created with GIMP`
+  ** **Property XPath** = `//*[local-name() = 'CreatorTool']`
+  ** **Property XPath** = `//*[starts-with(local-name(), 'Creator')]`
+
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:233]:
```suggestion
* The command line tool `exiftool` can be used to inspect a file for suitable metadata. 
* The output of `exiftool -n -q -b -X example.pdf` shows the metadata XML syntax. Use the XML element names to generate XPath queries and XML element data for the Property Value.
* The metadata may use XML namespaces. In `xmp:CreatorTool` the namespace is `xmp`. 
* We can use more complex XPath expressions to match the `CreatorTool` like in any namespace or by prefix: +
**Property XPath** = `//*[local-name()='Comment']` and **Property Value** = `Created with GIMP` +
**Property XPath** = `//*[local-name() = 'CreatorTool']` +
**Property XPath** = `//*[starts-with(local-name(), 'Creator')]`

Note that other XPath selectors than `local-name()` or `starts-with` are currently not supported.
```

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 5]
```diff
@@ -117,6 +117,18 @@ The configuration depends on the tools and the classification framework in use.
 
 Administrators can find examples and generalized configuration instructions below.
 
+=== EXIF Header Metadata
+
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:121]:
```suggestion
```

File: modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc [hunk 6]
```diff
@@ -183,6 +194,19 @@ This is how you set up an automated classification and the feature policy in own
 ** Hit btn:[Save].
 * To verify that the rule is in place, upload a classified file, check for the tag and try to create a public link share.
 
+===== EXIF Metadata
+
+The available elements in EXIF metadata can vary depending how the file was created.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/document_classification/classification_and_policy_enforcement.adoc:199]:
```suggestion
The available elements in EXIF metadata can vary depending how the file was created.

```

**Comment by jnweiger** [comment]
grafik

**Review by mmattel — APPROVED** [review]

**Pull request merged**